tests/test_common.h

```diff
@@ -76,6 +76,7 @@ mc::Client* newClient(int n) {
   mc::Client* client = new mc::Client();
   client->config(CFG_HASH_FUNCTION, OPT_HASH_MD5);
   client->init(hosts, ports, n, aliases);
+  // client->config(CFG_MAX_RETRIES, 1); // allow resend in waitPoll
```

Comment:
If we don't need it. I'd prefer to delete it.

Reply:
Sure. I left this to remind me that there's a `CFG_MAX_RETRIES` config, and when to invoke this - after client init.
But maybe it is useless for others. Better introduce it in docs.